src/server-glue/server.ts

```diff
@@ -35,7 +34,6 @@ if (!process.browser) {
 	const serveStatic: typeof ServeStatic = require('serve-static');
 	const cookieParser: typeof CookieParser = require('cookie-parser');
 	const bodyParser: typeof BodyParser = require('body-parser');
```

Comment:
Is this needed for the pinejs file upload? Will the WebResource.multerpineJS replace it for file upload that are not webresources?
```
const multer: typeof Multer = require('multer');
...
app.use(multer().any());

```

Reply:
@fisehara I moved the multer support to be applied only to models because I thought there were no uses for "file upload" and multer - there are no references to req.files or similar uses.

If we want to keep this, we can leave the app.use(multer().any()); here, and only put the handler ( webResource.handleMultipartRequest ) on the app models